.env.defaults

```diff
@@ -1,39 +1,66 @@
 #
-# See README for more info.
+# See README.md file for more info.
 #
 
-# Keycloak > Clients > "client name" > Installation > Keycloak OIDC JSON
-# .auth-server-url
-KEYCLOAK_SERVER=
-# .resource
-KEYCLOAK_RESOURCE=
-# .realm
-KEYCLOAK_REALM=
-# .credentials.secret
-KEYCLOAK_SECRET=
+################################################################################
+# Database connection configuration                                            #
+################################################################################
+RHUB_DB_TYPE=
+RHUB_DB_HOST=
+RHUB_DB_PORT=
+RHUB_DB_USERNAME=
+RHUB_DB_PASSWORD=
+RHUB_DB_DATABASE=
+
+################################################################################
+# Vault configuration                                                          #
+################################################################################
+# WARNING: Use file vault only for development!
+VAULT_TYPE=
+VAULT_PATH=
+# Tower webhook notification credentials path in Vault
+WEBHOOK_VAULT_PATH=
 
-# Amount of time that Dockerize will wait for Keycloak to become up and running
-KEYCLOAK_TIMEOUT="30s"
+################################################################################
+# PostgreSQL container's settings                                              #
+################################################################################
+POSTGRES_DB=
+POSTGRES_USER=
+POSTGRES_PASSWORD=
 
-# Admin user credentials
+################################################################################
+# Keycloak container's settings                                                #
+# Read more at: https://www.keycloak.org/getting-started/getting-started-docker#
+################################################################################
+# Keycloak > Clients > "client name" > Installation > Keycloak OIDC JSON
+# .credentials.secret
+DB_ADDR=
+DB_DATABASE=
+DB_HOST=
+DB_PASSWORD=
+DB_PORT=
+DB_USER=
+DB_USERNAME=
+DB_VENDOR=
 KEYCLOAK_ADMIN_USER=
 KEYCLOAK_ADMIN_PASS=
+KEYCLOAK_CLIENT=
+KEYCLOAK_IMPORT=
+KEYCLOAK_PASSWORD=
+KEYCLOAK_REALM=
+KEYCLOAK_RESOURCE=
+KEYCLOAK_SECRET=
+KEYCLOAK_SERVER=
+KEYCLOAK_USER=
 
-# Database configuration
-DB_TYPE=postgresql
-DB_HOST=localhost
-DB_PORT=5432
-DB_USERNAME=rhub
-DB_PASSWORD=rhub
-DB_DATABASE=rhub
-
-# Vault configuration
-# Use file vault only for development!
-VAULT_TYPE=file
-VAULT_PATH=data/vault.yml
-
-# Tower webhook notification credentials path in Vault
-WEBHOOK_VAULT_PATH=kv/webhookNotification
+################################################################################
+# Backend API                                                                  #
+################################################################################
+FLASK_APP=
+RHUB_SKIP_INIT=
+#RHUB_RETURN_INITIAL_FLASK_APP=
```

Comment:
Did you want this commented out?

Reply:
Yes :)
I removed it, since it's not necessary anymore at this point...